cypress/support/functions.ts

```diff
@@ -1053,7 +1056,15 @@ Cypress.Commands.add('epinioInstall', ({ s3, s3gw = false, extRegistry, namespac
   cy.contains('header', 'Charts', { timeout: 8000 }).should('be.visible');
 
   // Delete leftovers of other repos if existed.
-  cy.get('a[href="/dashboard/c/local/apps/catalog.cattle.io.app"] > span.count').then(($el) => {
+  // Ensure we are in local namespace to count only non-essenctial apps
```

Comment:
The `local namespace` is misleading here, actually the link points to `applications deployed on local cluster` (local cluster = the one running rancher mcm).